app/3.0/start/first-element/step-5.md

```diff
@@ -1,124 +1,152 @@
 
-demo-element.js: Before { .caption }
+demo/demo-element.js: Before { .caption }
 
 ```html
-    <style>
-      :host {
-        font-family: sans-serif;
-      }
-    </style>
+<style>
+  :host {
+    font-family: sans-serif;
+  }
+</style>
 ```
 
-demo-element.js: After { .caption }
+demo/demo-element.js: After { .caption }
 
 ```html
+<style>
+  :host {
+    font-family: sans-serif;
+    --icon-toggle-color: lightgrey;
+    --icon-toggle-outline-color: black;
+    --icon-toggle-pressed-color: red;
+  }
+</style>
+```
+
+Key information:
+
+  * Assign values to custom properties with a CSS rule. E.g.: 
+
+    ```css
     <style>
       :host {
-        font-family: sans-serif;
         --icon-toggle-color: lightgrey;
-        --icon-toggle-outline-color: black;
-        --icon-toggle-pressed-color: red;
       }
     </style>
-```
-
-Run the demo again to get colorful.
+    ```
 
+Run the demo again to get colorful!
 
-<img src="/images/3.0/first-element/toggles-styled.png" alt="Demo showing
-icon toggles with star and heart icons. Pressed icons are red.">
+<img src="/images/3.0/first-element/toggles-styled.png" alt="Demo showing icon toggles with star and heart icons. Pressed icons are red.">
 
-That's it — your element is finished. You've created an element that has a basic
-UI, API, and custom styling properties.
+That's it — your element is finished. You've created an element that has a basic UI, API, and custom styling properties.
 
-If you have any trouble getting the element working, check out the
-[finished version](https://github.com/PolymerLabs/polymer-3-first-element/tree/master/icon-toggle-finished).
+If you have any trouble getting the element working, the `icon-toggle-finished` folder contains the final code.
 
 If you'd like to learn a little more about custom properties, read on.
 
-## Extra credit: set custom properties at the document level
+### Extra credit: Set custom properties at the document level {#extra}
 
-Frequently you want to define custom property values at the document level, to
-set a theme for an entire application, for example. Because custom properties
-aren't built into most browsers yet, you need to use a special `custom-style`
-tag to define custom properties outside of a Polymer element. Try
-adding the following code inside the `<head>` tag of your `index.html` file:
+You might want to create a theme for an entire application. One way to do this is to define custom properties at the document level, outside of your Polymer elements. 
+
+Because custom properties aren't built into most browsers yet, if you use them outside a Polymer element, you need to use a special `custom-style` tag. 
+
+Add the following code inside the `<head>` tag of `demo/index.html`:
+
+demo/index.html {.caption}
 
 ```html
 <custom-style>
   <style>
-    /* Define a document-wide default—will not override a :host rule in  */
+    /* Define a document-wide default */
```

Comment:
These two comments should agree in tense, so either Defines/Overrides or Define / Override.